<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(254, 74, 73);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_calcular">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(254, 215, 102);
	border-radius: 15px;
	font: 75 10pt &quot;Perpetua&quot;;
	
}
QPushButton:hover{
	border:3px  solid rgb(254, 215, 102);
	
	color: rgb(255, 255, 255);
	background-color: rgb(0, 159, 183);
}</string>
    </property>
    <property name="text">
     <string>Calcular IMC</string>
    </property>
   </widget>
   <widget class="QLabel" name="Resultado">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(217, 219, 241);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="peso_txt">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(217, 219, 241);
border-radius:5px;</string>
    </property>
    <property name="placeholderText">
     <string>Ingrese su peso</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Altura_txt">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(217, 219, 241);
border-radius:5px;</string>
    </property>
    <property name="placeholderText">
     <string>Ingrese su altura</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
